app/views/components/tag.njk

```diff
@@ -0,0 +1,3 @@
+{% macro tag(qaIdentifier, text, classes) %}
```

Comment:
what do you mean by `qaIdentifier` here?

Reply:
it's just what we called it at my old work - I'll change it